packages/core/src/tools/create-lambda-handler.js

```diff
@@ -147,37 +147,36 @@ const createLambdaHandler = (appRawOrPath) => {
 
     environmentTools.cleanEnvironment();
 
+    // Copy bundle environment into process.env *before* creating the logger and
+    // loading app code, so that the logger gets the endpoint from process.env,
+    // and top level app code can get bundle environment vars via process.env.
+    environmentTools.applyEnvironment(event);
```

Comment:
I moved `applyEnvironment` up because I found on the logging endpoint could be different on the first invocation than the subsequent invocations.

[Here](https://github.com/zapier/zapier-platform/blob/5d6b06d39a149c600be6e7f838a0855c257f76ac/packages/core/src/tools/create-logger.js#L199) you can see the logging endpoint is either derived from the environment variable `LOGGING_ENDPOINT` or the constant `DEFAULT_LOGGING_HTTP_ENDPOINT`. But in the original (bad) code, we [`applyEnvironment`](https://github.com/zapier/zapier-platform/blob/5d6b06d39a149c600be6e7f838a0855c257f76ac/packages/core/src/tools/create-lambda-handler.js#L196) AFTER [`createLogger`](https://github.com/zapier/zapier-platform/blob/5d6b06d39a149c600be6e7f838a0855c257f76ac/packages/core/src/tools/create-lambda-handler.js#L152). So on the first invocation, it would get the logging endpoint from the constant. But on the second or subsequent calls, since the environment variables have been loaded, it gets the logging endpoint from the environment variable.

We don't see any issues on production because the constant and the environment variables are the same (`https://httplogger.zapier.com/input`) on production.

To fix, we need to make sure we `applyEnvironment` BEFORE `createLogger`.